VideoLocker/res/layout/row_course_list.xml

```diff
@@ -66,17 +66,8 @@
 
         <org.edx.mobile.view.custom.ETextView
             android:id="@+id/course_name"
-            style="@style/bold_grey_text"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginBottom="8dp"
-            android:layout_marginEnd="10dp"
-            android:layout_marginLeft="10dp"
-            android:layout_marginRight="10dp"
-            android:layout_marginStart="10dp"
-            android:layout_marginTop="10dp"
+            style="@style/course_card_title"
```

Comment:
All of these margins except bottom can be removed from the style as well, and applied to the parent's padding. Then the style can be applied to the `course_detail_name` view in `fragment_course_dashboard` as well.